### PR TITLE
Add Python 3 support

### PR DESCRIPTION
**Pull request by kosmos342**
- Added Python 3 support
- Reworked tests to use database
- Added testing on travis
- Added isort check

### CODE REVIEW

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

**Review by RayeN — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,7 +1,9 @@
 # [Changelog](https://github.com/yola/drf-madprops)
 
-## dev
+## 2.0.0
 * Dropped Python 2.6 support
+* Added Python 3 support
+* Add running tests Python 3.5-3.9, Django 1.11-3.2, djangorestframework 3.5-3.12
```
Comment (RayeN) [inline CHANGELOG.md:6]: Add -> Added

Comment (kosmos342) [reply CHANGELOG.md:6]: btw, do we need this info in the changelog?

Comment (RayeN) [reply CHANGELOG.md:6]: Yeah, good point - probably we don't

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,7 +3,7 @@
 ## 2.0.0
 * Dropped Python 2.6 support
 * Added Python 3 support
-* Add running tests Python 3.5-3.9, Django 1.11-3.2, djangorestframework 3.5-3.12
+* New version compatible with Django 1.11 - 3.2, DRF 3.5 - 3.12
```
Comment (RayeN) [inline CHANGELOG.md:6]: is compatible

**Review by RayeN — APPROVED** [review]

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

**Review by RayeN — APPROVED** [review]

**Pull request merged**